src/main/java/com/internet/shop/controllers/orders/DeleteOrderAdminController.java

```diff
@@ -0,0 +1,29 @@
+package com.internet.shop.controllers.orders;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Order;
+import com.internet.shop.service.OrderService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DeleteOrderAdminController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private final OrderService orderService =
+            (OrderService) injector.getInstance(OrderService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long id = Long.valueOf(req.getParameter("orderId"));
+
+        orderService.delete(id);
+        List<Order> orders = orderService.getAll();
+
+        req.setAttribute("orders", orders);
+        req.getRequestDispatcher("/WEB-INF/views/orders/admin.jsp").forward(req, resp);
```

Comment:
It seems to me you can resolve this with resp.sendRedirect - and then the list of orders will be updated and you don't need to repeat req.settAttribute, create "orders"...